src/map/skill.c

```diff
@@ -12814,7 +12899,7 @@ int skill_unit_onleft(uint16 skill_id, struct block_list *bl, int64 tick) {
 		case PF_FOGWALL:
 			if (sce) {
 				status_change_end(bl, type, INVALID_TIMER);
-				if ((sce=sc->data[SC_BLIND])) {
+				if (sc != NULL && (sce = sc->data[SC_BLIND])) {
```

Comment:
Side note, unrelated to this pull request - when you edit this, can you make it so `sce` is not overwritten (but use a separate variable instead)? Overwriting a variable from an outer scope to be used locally is really ugly...